src/main/java/com/example/java_pro_homework_7/Pet/Cat.java

```diff
@@ -0,0 +1,45 @@
+package com.example.java_pro_homework_7.Pet;
+
+public class Cat extends Animal {
+
+    public static int catsCounter;
+
+
+    public Cat(String name, String color, String breed) {
+        super(name, color, breed);
+        limitDistanceSwim = 0;
+        limitDistanceRun = 200;
+        catsCounter++;
+        getCatsCounter();
```

Comment:
You do not need to call `getCatsCounter();` from the constructor, please remove this call.